recipe/run_conda_forge_build_setup_osx

```diff
@@ -6,15 +6,13 @@ export CPU_COUNT=2
 
 export PYTHONUNBUFFERED=1
 
-export MACOSX_DEPLOYMENT_TARGET="10.9"
```

Comment:
This is removed here because it's in the conda_build_config.yaml instead, and gets carried through to each recipe's set of matrix files

Reply:
Is it alright if hang on to this a little longer? Mainly am concerned as there is no `conda_build_config.yaml` used by feedstocks currently. We can mark this as a `TODO` to remove after we have got most things re-rendered.

Is it still a passthrough environment variable with `conda-build` 3? If not, it may not matter.

Reply:
Sure.  It'll only get confusing when we decide to update the value next time, if we do it in conda_build_config.yaml and forget about it here.  I haven't changed the pass-throughs, so I'm pretty sure it still is one.

Reply:
That's fine. I'm hoping by then we will already have dropped this. If we haven't, that will mark a good time to finally drop it.

Reply:
restored, with TODO note.